python/packages/autogen-core/src/autogen_core/tools/_base.py

```diff
@@ -37,6 +37,9 @@ def description(self) -> str: ...
     @property
     def schema(self) -> ToolSchema: ...
 
+    @property
+    def return_errors(self) -> tuple[type[Exception], ...] | type[Exception]: ...
```

Comment:
Let's rename it `return_error_types`? This makes it align with the `return_type` property that already exists. 

Let's also add API doc comment to each field of `Tool` and `BaseTool` as a good Samaritan 😄

Reply:
> Let's also add API doc comment to each field of Tool and BaseTool as a good Samaritan

Not sure what you mean by this. Do you want me to add docstrings to every function in Tool/BaseTool?

Reply:
Yes. I was asking if you would be willing to add the docstrings.